pkgs/os-specific/linux/sysdig/default.nix

```diff
@@ -3,14 +3,28 @@ let
   inherit (stdenv.lib) optional optionalString;
   s = rec {
     baseName="sysdig";
-    version = "0.10.0";
+    version = "0.11.0";
     name="${baseName}-${version}";
     url="https://github.com/draios/sysdig/archive/${version}.tar.gz";
-    sha256 = "0hs0r9z9j7padqdcj69bwx52iw6gvdl0w322qwivpv12j3prcpsj";
+    sha256 = "0fqh8vz5an3bw501y7h05l74pfv1ha88mm1k3prh0gdkiqf6dbjr";
   };
   buildInputs = [
     cmake zlib luajit ncurses perl jsoncpp libb64 openssl curl
   ];
+  # sysdig-0.11.0 depends on some headers from jq which are not
+  # installed by it.  Sysdig allows the use of a bundled jq version,
+  # from which it would normally get the headers, but this requires
+  # network during the build which is not available with chroot in
+  # nix.
+  #
+  # As a workaround, we enable the bundled jq from sysdig, but
+  # download the jq archive prior to the build, such that it can work
+  # under chroot.
+  # Relevant sysdig issue: https://github.com/draios/sysdig/issues/626
```

Comment:
I think so. Just mentioning the header issue would be sufficient, though (that builds cannot have network access is implied by purity, even if enforcement is optional).